multistream-select/src/protocol/error.rs

```diff
@@ -38,6 +39,10 @@ pub enum MultistreamSelectError {
 
 	/// Protocol names must always start with `/`, otherwise this error is returned.
 	WrongProtocolName,
+
+	/// Failure to parse variable-length integer.
+	// TODO: we don't include the actual error, because that would remove Send from the enum
+	VarIntParseError,
```

Comment:
`VarintParseError`, also maybe we should include a `String` field and populate it with `format!("{}", error)`. It's ugly but it's better than nothing.